libraries/hardwareInterfaces.js

```diff
@@ -849,8 +849,8 @@ exports.setHardwareInterfaceSettings = function (interfaceName, settings, limitT
  * Complement to setup() which is necessary due to the unique positioning of
  * the setHardwareInterfaceSettings function
  */
-exports.setHardwareInterfaceSettingsImpl = function (setHardwareInterfaceSettings) {
-    setHardwareInterfaceSettingsImpl = setHardwareInterfaceSettings;
+exports.setSetHardwareInterfaceSettingsImpl = function (newSetHardwareInterfaceSettingsImpl) {
```

Comment:
Not the biggest fan of `setSet`, but I see what's going on here

Reply:
[8258313](https://github.com/ptcrealitylab/vuforia-spatial-edge-server/pull/907/commits/825831351f339b8e9242a8c483b806275d5d4b53) has good news for you